vpr/src/route/connection_router.cpp

```diff
@@ -477,27 +535,55 @@ void ConnectionRouter<Heap>::timing_driven_expand_neighbour(t_heap* current,
     VTR_LOGV_DEBUG(router_debug_, "      Expanding node %d edge %zu -> %d\n",
                    from_node, size_t(from_edge), to_node_int);
 
-    timing_driven_add_to_heap(cost_params,
-                              current,
-                              from_node,
-                              to_node_int,
-                              from_edge,
-                              target_node);
+    // Check if the node exists in the route tree, needed for RCV code
```

Comment:
This comment is insufficient to explain why the router expansion is being modified in this way.

Reply:
That's fair I'll write a better comment for this.